index.html

```diff
@@ -0,0 +1,2294 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Title</title>
+    <!-- Required meta tags -->
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+
+    <!-- Bootstrap CSS -->
+
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+      integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css"
+    />
+    <link rel="stylesheet" href="assets/css/style.css" />
+  </head>
+  <body>
+    <nav class="navbar fixed-top red-back">
```

Comment:
- [ ] Kindly fix the navbar for large screens. At least in my end, it has differences. I also notice that the link provided on the PR is from a previous commit, kindly update it.

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83579681-8c8eb580-a4ff-11ea-9c20-3ac11492b8d8.png) | ![image](https://user-images.githubusercontent.com/57414789/83579702-987a7780-a4ff-11ea-82c9-0b195c23cbad.png)